### ISSUE TITLE
Raspberry Pi

### ISSUE THREAD

**Issue by akorittko** [TheBerkin/cards-over-lan#8]
How can I run this from a raspberry pi

**Comment by TheBerkin** [comment]
I would try using Mono to run this on a Pi, assuming you're running some kind of Linux.

(Disclaimer: I haven't tried this on a Pi myself, but I have successfully deployed CoLAN over Mono on a Debian VPS before.)

**Comment by akorittko** [comment]
Ok will look into that.... was just asking because the README.md mentioned it. :) 
![CAHgit](https://user-images.githubusercontent.com/58423/83708686-d902ef80-a5ea-11ea-9bf4-ddc7489f42ef.png)

**Comment by TheBerkin** [comment]
Yep, Raspberry Pi OS is based on Debian so I see no reason why it wouldn't work. If you run into any compatibility problems, though, I'd be happy to investigate.
